### PR TITLE
Add X-team info

### PR DESCRIPTION
**Pull request by arieljimenez**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [ ] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [ ] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/x-team.md [hunk 1]
```diff
@@ -2,10 +2,29 @@
...
-⚠ We don't have much information about this company yet!
+Since 2006, X-Team has provided trusted, motivated developers to companies looking to scale their teams.
 
-If you know something we don't, help us fill it in!  Here's how:
+## Company size
 
-- Read our [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md)
-- Have a look at our [example company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md)
-- Follow the structure of the example profile and [send us a pull request with your changes to this file!](https://github.com/remoteintech/remote-jobs/edit/master/company-profiles/x-team.md)
+100-200
```
Comment (arieljimenez) [inline company-profiles/x-team.md]: I couldn't find the exact number on their page, so I did a look up in linkedin and found that 176 people said that work there.

Comment (nylen) [reply company-profiles/x-team.md]: Maybe someone who works there will come along and update this. Until then, this is fine :+1:

**Comment by nylen** [comment]
This PR looks good, but we were having an issue with the build (fixed in #662). If you can merge or rebase against the latest code from our `master` branch then we can get these changes in.

**Comment by arieljimenez** [comment]
done @nylen

**Pull request merged**